README.md

```diff
@@ -194,6 +194,20 @@ With the [`ini_setting` module](https://forge.puppet.com/modules/puppetlabs/inif
 
 5. Verify your Puppet data is in Datadog by searching for `sources:puppet` in the [Event Stream][5].
 
+### NPM setup
+
+To enable the Datadog Agent Network Performance Monitoring (NPM) features follow these steps:
+
+1. (Windows only) If the Agent is already installed, uninstall it by passing `win_ensure => absent` to the main class.
```

Comment:
This fails if the `datadog_agent::system_probe` class is defined, with:
```
Error: Could not find resource 'File[C:/ProgramData/Datadog]' in parameter 'require' (file: C:/ProgramData/PuppetLabs/code/environments/production/modules/datadog_agent/manifests/system_probe.pp, line: 35) on node <node name>
```
so we should also mention that you also need to remove the `datadog_agent::system_probe` class.

Reply:
```suggestion
1. (Windows only) If the Agent is already installed, uninstall it by passing `win_ensure => absent` to the main class and removing other classes.
```